### PR TITLE
Add test 'Check mailto link on contact us page'

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.7.x
| Description?      | Add test 'Check mailto link on contact us page'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | `TEST_PATH='function/FO/04_contactUs/01_checkMailtoLink.js' URL_FO=shopUrl/ npm run specific-test`
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23679)

### CODE REVIEW

File: tests/UI/campaigns/functional/FO/04_contactUs/01_checkMailtoLInk.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+const homePage = require('@pages/FO/home');
+const contactUsPage = require('@pages/FO/contactUs');
+
+// Import data
+const {DefaultEmployee} = require('@data/demo/employees');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_FO_headerAndFooter_checkLinksInHeader';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/04_contactUs/01_checkMailtoLInk.js]:
```suggestion
const baseContext = 'functional_FO_contactUs_checkMailtoLink';
```

:warning: @boubkerbribri the name of the file has LInk and not Link ;)

**Review by Progi1984 — COMMENTED** [review]
Small feedbacks

File: tests/UI/campaigns/functional/FO/04_contactUs/01_checkMailtoLInk.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/campaigns/functional/FO/04_contactUs/01_checkMailtoLInk.js]: done thanks

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @boubkerbribri

**Pull request merged**